packages/components/src/IconItem/IconItem.scss

```
@@ -30,6 +30,14 @@
     @include iconitem($color-success-light);
   }
 
+  .#{$prefix}--icon-item__icon--blue-light {
+    @include iconitem($color-blue-dark-lighter);
+  }
```

Acá para ser consistente con las otras appearances tendría que ser `secondary-light` y el nombre de la variable de sass es `$color-secondary-lighter`

```suggestion
  .#{$prefix}--icon-item__icon--secondary-light {
    @include iconitem($color-secondary-lighter);
  }
```

---

Genial tiene mucho sentido, ahi lo cambié